synth.py

```diff
@@ -6,30 +6,49 @@
 logging.basicConfig(level=logging.DEBUG)
 
 # Run the gapic generator
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 version = 'v1'
-library = gapic.node_library(
-    'container', version,
-    config_path="/google/container/"
-                "artman_container_v1.yaml")
+library = gapic.typescript_library(
+    'container',
+    generator_args={
+            "grpc-service-config": f"google/container/{version}/container_grpc_service_config.json",
+            "package-name": f"@google-cloud/container"
+            },
+            proto_path=f'/google/container/{version}',
+            version=version)
 s.copy(
     library,
-    excludes=['package.json', 'README.md', 'src/index.js'],
+    excludes=['package.json', 'README.md', 'src/index.ts'],
 )
 
 # Copy templated files
 common_templates = gcp.CommonTemplates()
-templates = common_templates.node_library()
+templates = common_templates.node_library(source_location='build/src')
 s.copy(templates)
 
-s.replace("src/v1/doc/google/container/v1/doc_cluster_service.js",
-        "<a href=\"\/compute\/docs\/resource-quotas\">resource quota<\/a>",
-        r"[resource quota](https://cloud.google.com/compute/docs/resource-quotas)")
+# fix broken doc links
 
 s.replace("src/v1/doc/google/container/v1/doc_cluster_service.js",
         "https:\/\/cloud\.google\.com\/kubernetes-engine\/docs\/reference\/rest\/v1\/projects\.zones\.clusters\.nodePool",
         "https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.zones.clusters.nodePools#resource-nodepool")
 
+s.replace('src/v1/*.ts',
+        '/compute/docs/zones#available',
+        'https://cloud.google.com/compute/docs/regions-zones/')
```

Comment:
the `#available` is lost here, do they have this anchor?

Reply:
rechecked the ts file, it only contains `/compute/docs/zones` one replace should work:
```
s.replace('src/v1/*.ts',
        '/compute/docs/zones',
        'https://cloud.google.com/compute/docs/regions-zones/')
```